src/Firehose.Web/Infrastructure/CombinedFeedSource.cs

```diff
@@ -61,7 +61,15 @@ private static ISyndicationFeedSource TryLoadFeed(IAmACommunityMember tamarin, U
                     .FromUrl(uri)
                     .WithFilter(filter)
                     .Grr();
-                var dummy = feedSource.Feed; // poke it to make sure it doesn't go bang.
+
+                // Poke it, to see if we can really reach it
+                var request = (HttpWebRequest)WebRequest.Create(uri);
+                request.Method = "HEAD";
+
+                var response = (HttpWebResponse)request.GetResponse();
+
+                if (response.StatusCode != HttpStatusCode.OK)
```

Comment:
I think this should check status codes are between 200 and 399

Redirects are valid too, no?